schemas/tls.yml

```diff
@@ -0,0 +1,189 @@
+---
+- name: tls
+  title: TLS
+  group: 2
+  short: Fields describing a TLS connection.
+  description: >
+    Fields related to TLS activity.
+  type: group
+  fields:
+
+    - name: version
+      level: extended
+      type: group
+      description: Fields related to the SSL/TLS version used.
+      fields:
+        - name: original
```

Comment:
I wonder how useful this one will be since we have a normalized protocol name and version number. When information is coming from a log we will still have the original value in the `message` (albeit unparsed). When the info is coming directly from network traffic there isn't really a value for this unless we write some bytes (e.g. `0x303`). Perhaps we don't need this field.

Reply:
That's a fair point. I just noticed that Suricata, Zeek, and Packetbeat all provide a differently formatted value for this field, but if that's being preserved in `message`, that's probably good enough.